### DISCUSSION TITLE
Feature request: Trigger a deployment from the portal

### DISCUSSION THREAD

**Discussion by skttl** [umbraco/Umbraco.Cloud.Issues#792] [Ideas]
### Issue description

Sometimes it's nice to be able to trigger a deployment without having to push code to the repository. I just learned that you can do a Postman PUT to https://{your-site-alias}.scm.euwest01.umbraco.io/api/deployments with basic auth using your credentials for umbraco.io (no body), to trigger a deployment. Thats cool! If it was a button in the portal it would be even cooler!

**Comment by sajumb** [comment]
Thanks @skttl. Great suggestion. I have added this feature request to our backlog.
